icu4c/source/common/unicode/uversion.h

```diff
@@ -63,25 +63,41 @@ typedef uint8_t UVersionInfo[U_MAX_VERSION_LENGTH];
 /*===========================================================================*/
 
 /**
- * \def U_NAMESPACE_BEGIN
- * This is used to begin a declaration of a public ICU C++ API.
+ * \def U_CXX_BEGIN
+ * This is used to begin a C++ block.
  * When not compiling for C++, it does nothing.
  * When compiling for C++, it begins an extern "C++" linkage block (to protect
  * against cases in which an external client includes ICU header files inside
  * an extern "C" linkage block).
  *
- * It also begins a versioned-ICU-namespace block.
+ * @stable ICU 6.5
```

Comment:
Is this better? If not, please instruct what to put.

Reply:
`@stable ICU 65`

Reply:
So are we good? I see all over that there is a dot between major/minor.

Reply:
You may not include `@stable 65`. If proposed as an API it would be `@draft 65`